src/tests/ftest/container/SnapshotInfo.py

```diff
@@ -0,0 +1,550 @@
+                             createsize, createsetid, None)
+
+            # need a connection to create container
+            self.pool.connect(1 << 1)
+
+            # create a container
+            self.log.info("==Creating a container")
+            self.container = DaosContainer(self.context)
+            self.container.create(self.pool.handle)
+
+            # now open it
+            self.log.info("==Open the container")
+            self.container.open()
+
+        except DaosApiError as error:
+            self.log.info(error)
+            self.log.info(traceback.format_exc())
+            self.fail("##Test failed before snapshot taken")
+
+    def tearDown(self):
+        try:
+            if self.container:
+                self.log.info("==tearDown, close and destroy the container.")
+                self.container.close()
+                self.container.destroy()
+            if self.pool:
+                self.log.info("==tearDown, disconnect and destroy the pool.")
+                self.pool.disconnect()
+                self.pool.destroy(1)
+        finally:
+            self.log.info("==tearDown, stop the DAOS server.")
+            ServerUtils.stopServer()
+
+    def display_DaosContainer(self):
+        self.log.info("==display_DaosContainer========================")
+        self.log.info("self.container OBJ=                {0}"\
+            .format(self.container))
+        self.log.info("self.container.context obj=        {0}"\
+            .format(self.container.context))
+        self.log.info("self.container.context.libdaos=    {0}"\
+            .format(self.container.context.libdaos))
+        self.log.info("self.container.context.libtest=    {0}"\
+            .format(self.container.context.libtest))
+        self.log.info("self.container.context.ftable=     {0}"\
+            .format(self.container.context.ftable))
+        self.log.info("self.container.uuid obj=           {0}"\
+            .format(self.container.uuid))
+        self.log.info("self.container.coh=                {0}"\
+            .format(self.container.coh))
+        self.log.info("self.container.poh=                {0}"\
+            .format(self.container.poh))
+        self.log.info("self.container.info obj=           {0}"\
+            .format(self.container.info))
+        self.log.info("self.container.info.ci_uuid=       {0}"\
+            .format(self.container.info.ci_uuid))
+        self.log.info("self.container.info.es_hce=        {0}"\
+            .format(self.container.info.es_hce))
+        self.log.info("self.container.info.es_lre=        {0}"\
+            .format(self.container.info.es_lre))
+        self.log.info("self.container.info.es_lhe=        {0}"\
+            .format(self.container.info.es_lhe))
+        self.log.info("self.container.info.es_ghce=       {0}"\
+            .format(self.container.info.es_ghce))
+        self.log.info("self.container.info.es_glre=       {0}"\
```

Comment:
(pylint-logging-format-interpolation) Use % formatting in logging functions and pass the % parameters as arguments